common/src/main/java/io/netty/util/concurrent/EventExecutorGroup.java

```diff
@@ -116,4 +110,12 @@
 
     @Override
     ScheduledFuture<?> scheduleWithFixedDelay(Runnable command, long initialDelay, long delay, TimeUnit unit);
+
+    /**
+     * {@link EventExecutorGroup} can be wrapped to extend and enhance behavior. This method will unroll the wrapping
+     * and return the underlying {@link EventExecutorGroup}.
+     *
+     * @return {@code this} if the object is not wrapped or underlying {@link EventExecutorGroup} instance if wrapped.
+     */
+    EventExecutorGroup unRollWrapping();
```

Comment:
Comment was collapsed, but still applicable:

> @nmittler @normanmaurer - Can you help me think of a better name?  I am torn because I don't want to use `unwrap` because this interface is *slightly* different than [ByteBuf.unwrap()](https://github.com/netty/netty/blob/4.1/buffer/src/main/java/io/netty/buffer/ByteBuf.java#L287). This method does not return `null` if the object is not wrapped (like `ByteBuf.unwrap()`), but instead always returns the "unwrapped" object. The difference is mainly because I don't understand the benefits of returning `null` for an interface like this, and for the `EventExecutorGroup` type hierarchy it would add additional conditional checks and complicate the code.